src/sqlalchemy_declarative_extensions/dialects/postgresql/trigger.py

```diff
@@ -87,6 +87,7 @@ class Trigger(base.Trigger):
     time: TriggerTimes
     for_each: TriggerForEach = TriggerForEach.statement
     condition: str | None = None
+    arguments: str | None = None
```

Comment:
should this be

```suggestion
    arguments: list[str] | None = None
```

? The docs for [create trigger](https://www.postgresql.org/docs/current/sql-createtrigger.html) imply it'll be a comma delimited list of arguments.

Although i do see that `tgargs` is a byta, which i assume means there'll be some parse/mapping required to get it to/from the comma delimited list in sql syntax

Reply:
You're right, I misunderstood that first. Fixed in today's commit.